pkgs/native_assets_builder/test_data/treeshaking_native_assets/.gitignore

```diff
@@ -0,0 +1,3 @@
+# https://dart.dev/guides/libraries/private-files
+# Created by `dart pub`
+.dart_tool/
```

Comment:
Ignore .so .dylib .dll and .exe files (or ignore the folder in bin/ that will be generated by `dart build`)

Great to see that it works on your end!  🎉